src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -705,7 +705,10 @@ def _get_private_clauses(self):
                     if loop_ancestor:
                         # The assignment to the variable is inside a loop, so
                         # declare it to be private
-                        name = str(signature).lower()
+                        if str(signature).find("%") > 0:
```

Comment:
I just noticed this: this is the same as `signature[0].lower()` (unless I am missing something)

Reply:
Ok I'll swap it to that.